search/linear_search.cpp

```diff
@@ -63,30 +67,42 @@ static void tests() {
  * @brief Main function
  * @returns 0 on exit
  */
-int main() {
+int main()
+{
     int mode = 0;
 
     std::cout << "Choose mode\n";
     std::cout << "Self-test mode (1), interactive mode (2): ";
 
     std::cin >> mode;
 
-    if (mode == 2) {
+    if (mode == 2)
+    {
         int size = 0;
-        std::cout << "\nEnter the size of the array: ";
+        std::cout << "\nEnter the size of the array [in range 1-30 ]: ";
         std::cin >> size;
 
-        while ((size <= 1) || (size >= 30)) {
-            std::cout << "Size cannot be less than zero. Please choose another value: ";
-            std::cin >> size;
+        while (size <= 0 || size > 30)
```

Comment:
Hmm, I'm not sure what's the difference here. Could you explain the benefits, please?

Reply:
Previously user was prompted to enter array size again even if it was greater than 0 (but it prompted "size cannot be less than 0"), because the user may have entered the size greater than 30. So, for the "user's clarity" regarding the input, the prompt should be made accordingly.